Voorbeelden bij implementatietoelichting onzelfstandig ReadTheDocs (#193)
woonstadrotterdam/woningwaardering: docs/implementatietoelichtingen/zelfstandige-woonruimten.md

### Rubriek 11 Punten voor de WOZ-waarde

> [!TIP]

> > Hieronder vindt u een voorbeeld hoe punten voor de WOZ-waarde berekend kunnen worden.

Reviewer: ```suggestion
> Dit voorbeeld toont hoe punten voor de WOZ-waarde berekend kunnen worden.
```

**~~Woningen gelegen in COROP-gebied (2018-2022)~~**

> ~~Indien de bouwkundige oplevering van een woning, die kleiner is dan 40 m2, heeft plaatsgevonden in de jaren 2018-2022 en de woning is gelegen in de COROP-gebieden5 Amsterdam en Utrecht, dan wordt het puntenaantal voor de WOZ-waarde berekend op basis van een ander kengetal dan wat in de paragraaf hierboven is weergegeven. In het tweede onderdeel van de berekening wordt de WOZ-waarde gedeeld door het aantal m2 van de oppervlaktes van de woning, gedeeld door het kengetal € 242 (peildatum 1 januari 2024). Bij kleine nieuwbouwwoningen (<40 m2) gelegen in COROP-regio’s Amsterdam en Utrecht wordt dan niet het kengetal € 242 gebruikt maar € 103 bij de peildatum 1 januari 2024. Bij de peildatum 1 januari 2023 wordt niet het bedrag van € 229 gebruikt, maar € 97.~~

Reviewer: voetnoot?

Reviewer 2: ```suggestion
~~Indien de bouwkundige oplevering van een woning, die kleiner is dan 40 m2, heeft plaatsgevonden in de jaren 2018-2022 en de woning is gelegen in de COROP-gebieden[^7] Amsterdam en Utrecht, dan wordt het puntenaantal voor de WOZ-waarde berekend op basis van een ander kengetal dan wat in de paragraaf hierboven is weergegeven. In het tweede onderdeel van de berekening wordt de WOZ-waarde gedeeld door het aantal m2 van de oppervlaktes van de woning, gedeeld door het kengetal € 242 (peildatum 1 januari 2024). Bij kleine nieuwbouwwoningen (<40 m2) gelegen in COROP-regio’s Amsterdam en Utrecht wordt dan niet het kengetal € 242 gebruikt maar € 103 bij de peildatum 1 januari 2024. Bij de peildatum 1 januari 2023 wordt niet het bedrag van € 229 gebruikt, maar € 97.~~
```

> ~~Indien de bouwkundige oplevering van een woning, die kleiner is dan 40 m2, heeft plaatsgevonden in de jaren 2018-2022 en de woning is gelegen in de COROP-gebieden5 Amsterdam en Utrecht, dan wordt het puntenaantal voor de WOZ-waarde berekend op basis van een ander kengetal dan wat in de paragraaf hierboven is weergegeven. In het tweede onderdeel van de berekening wordt de WOZ-waarde gedeeld door het aantal m2 van de oppervlaktes van de woning, gedeeld door het kengetal € 242 (peildatum 1 januari 2024). Bij kleine nieuwbouwwoningen (<40 m2) gelegen in COROP-regio’s Amsterdam en Utrecht wordt dan niet het kengetal € 242 gebruikt maar € 103 bij de peildatum 1 januari 2024. Bij de peildatum 1 januari 2023 wordt niet het bedrag van € 229 gebruikt, maar € 97.~~

Reviewer: ```suggestion
~~Indien de bouwkundige oplevering van een woning, die kleiner is dan 40 m2, heeft plaatsgevonden in de jaren 2018-2022 en de woning is gelegen in de COROP-gebieden Amsterdam en Utrecht, dan wordt het puntenaantal voor de WOZ-waarde berekend op basis van een ander kengetal dan wat in de paragraaf hierboven is weergegeven. In het tweede onderdeel van de berekening wordt de WOZ-waarde gedeeld door het aantal m2 van de oppervlaktes van de woning, gedeeld door het kengetal € 242 (peildatum 1 januari 2024). Bij kleine nieuwbouwwoningen (<40 m2) gelegen in COROP-regio’s Amsterdam en Utrecht wordt dan niet het kengetal € 242 gebruikt maar € 103 bij de peildatum 1 januari 2024. Bij de peildatum 1 januari 2023 wordt niet het bedrag van € 229 gebruikt, maar € 97.~~
```
